python/cuml/neighbors/nearest_neighbors_mg.pyx

```diff
@@ -28,6 +28,7 @@ from cuml.neighbors import NearestNeighbors
 from pylibraft.common.handle cimport handle_t
 from cuml.common.opg_data_utils_mg cimport *
 from cuml.common.opg_data_utils_mg import _build_part_inputs
+from cuml.metrics.distance_type cimport DistanceType
```

Comment:
pylibraft provides this (and also provides the enum) so we can cut down on redundancy here.